### ISSUE TITLE
The contrast of text against its background does not meet minimum contrast requirements

### ISSUE THREAD

**Issue by guybark** [Microsoft/BotFramework-WebChat#1532]
Run the Keros for Desktop tool and point its color contrast checker to the light gray text in an Adaptive Card. The attached image shows the tool reporting a failure, as the contrast of the text against its background does not meet the minimum requirements of 4.5:1.

**Comment by guybark** [comment]
Adding the Keros for Desktop screenshot...

![webchatcontrast](https://user-images.githubusercontent.com/14315611/50358114-1b2a9500-050d-11e9-85ce-6dd48b2d2565.png)

**Comment by corinagum** [comment]
@guybark Thanks for filing! Did you intend to close this issue? I'm happy to keep it open until we can confirm the accessibility fixes on your updated bot. :) Please let me know if I should reopen!
